english/bear-story/bear-story.md

```diff
@@ -0,0 +1 @@
+"[Yuna](https://xantandminions.wordpress.com/kuma-kuma-kuma-bear/). Nice to meet you," she says, as she dances around in her bear dress and summons two giant bears to haul you away to another fantasy world as her bear slave. You end up in the body of a 10-year-old and are forced to participate in the bear takeover of the world!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.